core/dbt/tests/fixtures/project.py

```diff
@@ -63,6 +63,19 @@ def test_data_dir(request):
     return os.path.join(request.fspath.dirname, "data")
 
 
+@pytest.fixture(scope="class")
+def dbt_profile_target():
```

Comment:
I added `profiles_config_update ` to do similar stuff, do we want to consider merging these two functions?

Reply:
The profile_target is for just the important bits and is consistent with the way adapter tests ran on dbt-adapter-tests before. The profiles_config_update is for when you need to replace more than the "target" portion. So I think they serve different needs. There's overlap, certainly...  If it turns out to be confusing or a headache we can revisit.